skulpt.py

```diff
@@ -753,9 +757,10 @@ def dist(options):
     doc()
 
     try:
-        shutil.copy(compfn, os.path.join(DIST_DIR, "tmp.js"))
```

Comment:
Somewhere the build script relies on this file existing at some point, could you add these lines back in?